app-server/migrations/src_migrations/main/scala/10.scala

```diff
@@ -0,0 +1,29 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M10 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(10)(List(
+    sqlu"""
+CREATE TABLE buckets (
+  id UUID PRIMARY KEY NOT NULL,
+  created_at TIMESTAMP NOT NULL,
+  modified_at TIMESTAMP NOT NULL,
+  organization_id UUID REFERENCES organizations(id) NOT NULL,
+  created_by VARCHAR(255) REFERENCES users(id) NOT NULL,
+  modified_by VARCHAR(255) REFERENCES users(id) NOT NULL,
+  name TEXT NOT NULL,
```

Comment:
That was for having a primary key that was a `text` field. For description/name that are not going to be indexed (without splitting and doing things to support full text search) I think we want to allow undefined amounts of text for these.